requests/pulls and committs.py

```diff
@@ -0,0 +1,85 @@
+import requests
```

Comment:
Your pull request REQUESTS-1022 Added pulls and commits.py/nThe prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']

Your commit: REQUESTS-1022 Added pulls and committs.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: REQUESTS-1022 Refactored pulls and committs.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: REQUESTS-1022 Refactored pulls and commits.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: REQUESTS-1022 Refactored pulls and commits.py
The prefix of your message must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']
Your commit: Update pulls and committs.py
You need to write "-" between prefix of your message and group number